ray-operator/controllers/ray/raycluster_controller.go

```diff
@@ -121,27 +123,42 @@ func (r *RayClusterReconciler) eventReconcile(request ctrl.Request, event *corev
 	// we only care about pod events
 	if event.InvolvedObject.Kind != "Pod" || event.Type != "Warning" || event.Reason != "Unhealthy" ||
 		!strings.Contains(event.Message, "Readiness probe failed") {
+		// This is not supposed to happen
+		r.Log.Error(fmt.Errorf("unexpected event"), "event", event)
```

Comment:
em. it means we log all other unrelated events which is unnecessary? Let's say there're some other components throw events, it will be printed in kuberay controller log. I feel we should remove this line